modules/33_CDR/portfolio/equations.gms

```diff
@@ -34,42 +34,47 @@ q33_emiCDR(t,regi)..
     ;
```

Comment:
add emissions not captured here, so that this is the "net CDR". Would work the same way for DAC